terms.html

```diff
@@ -116,7 +134,7 @@
   </dd>
   <dt><dfn data-lt="subjects|subject's">subject</dfn></dt>
   <dd>
-A thing about which <a>claims</a> are made.
+The <a>entity</a> about which a <a>claims</a> is made.
```

Comment:
This either needs to be made plural or singular, right now it's a combination of the two:

```suggestion
The <a>entity</a> about which <a>claims</a> are made.
```

Reply:
I think it makes more sense in context if its singular...
```suggestion
The <a>entity</a> about which a <a>claim</a> is made.
```

Reply:
agree with @TallTed